spell/Frog God Games; Book of Lost Spells.json

```diff
@@ -6113,7 +6113,7 @@
 			},
 			"entries": [
 				"You summon the spirit of a powerful, once-living champion of your deity and give it physical form. The spirit champion arrives within range, looking as it did in the prime of its life. The spell must be cast at a site hallowed to the caster's deity.",
-				"The spirit champion possesses all abilities and items that it possessed during life. In addition, the spirit champion is able to cast command or suggestion as an action on any worshipper of the caster's deity. Its exact statistics are determined by the GM; we suggest using the planetar's stat block as a starting point, but the specifics are entirely up to the GM. It won't give or loan items to other characters.",
+				"The spirit champion possesses all abilities and items that it possessed during life. In addition, the spirit champion is able to cast command or suggestion as an action on any worshipper of the caster's deity. Its exact statistics are determined by the GM; we suggest using the {@creature planetar's|planetar} stat block as a starting point, but the specifics are entirely up to the GM. It won't give or loan items to other characters.",
```

Comment:
`{@creature planetar's|planetar}` -> `{@creature planetar||planetar's}`
note that the `|<source>|` bit can just be blank if it's a MM creature